test/images/agnhost/dns/common.go

```diff
@@ -76,6 +76,7 @@ func readFile(fileName string) string {
 	return string(fileData)
 }
 
+//lint:ignore U1000 false positive, used in test/images/agnhost/dns/dns_windows.go
```

Comment:
If the function is only used by the Windows build, wouldn't it make sense to move it to dns_windows.go instead of keeping it in common.go?

Reply:
It does. Thanks!